src/Impostor.Server/Net/State/Game.State.cs

```diff
@@ -65,6 +65,7 @@ private async ValueTask<bool> PlayerRemove(int playerId, bool isBan = false)
             if (HostId == playerId)
             {
                 await MigrateHost();
+                await _eventManager.CallAsync(new GameHostChangedEvent(this, player, this.Host));
```

Comment:
also explicit `this.` isn't used in this project, so change `this.Host` to just `Host`